terraform_compliance/steps/given/i_have_name_section_configured.py

```diff
@@ -41,6 +41,8 @@ def i_have_name_section_configured(_step_obj, name, type_name='resource', _terra
 
     if not hasattr(_step_obj.context, 'cumulative_stash'):
         _step_obj.context.cumulative_stash = []
+    if not hasattr(_step_obj.context, 'stash'):
+        _step_obj.context.stash = []
```

Comment:
This change is not necessary. Existence of the field can be checked in terrain.py

I thought it would be a better idea to make sure that a stash is always created the Given step